testdata/p4tc_samples_outputs/direct_counter_example.json

```diff
@@ -4,7 +4,7 @@
   "externs" : [
     {
       "name" : "DirectCounter",
-      "id" : 101,
+      "id" : 1,
```

Comment:
Why this is changed, though there is no change in code

Reply:
addressed.